README.md

```diff
@@ -131,3 +131,14 @@ There you will find Express.js, TypeScript and Websocket examples.
 We love contributions from the community! Please check the
 [Contributor's Guide](https://github.com/cloudevents/sdk-javascript/blob/master/CONTRIBUTING.md)
 for information on how to get involved.
+
+While there are some SDK specific processes defined, most governance related
+material can be found in the
+[CloudEvents `community`](https://github.com/cloudevents/spec/tree/master/community)
+directory. In particular, in there you will find information concerning
+how SDK projects are
+[managed](https://github.com/cloudevents/spec/blob/master/community/SDK-GOVERNANCE.md),
+[guidelines](https://github.com/cloudevents/spec/blob/master/community/SDK-maintainer-guidelines.md)
+for how PRs are reviewed, and our
+[Code of Conduct](https://github.com/cloudevents/spec/blob/master/community/GOVERNANCE.md#additional-information)
```

Comment:
Should this just link directly to the CNCF CoC?

Reply:
I decided for the indirect ref in the hopes that if we produce our own CoC one day then we'd only need to change the spec's repo and not each individual SDK's.  But if you feel we should point directly to CNCF's I can change it.